### ISSUE TITLE
Fallback to home page 

### ISSUE THREAD

**Issue by VorpalBlade** [M4SS-Code/cargo-goggles#6]
```
Package tiny-keccak is missing `repository` attribute in Cargo.toml
Package crunchy is missing `repository` attribute in Cargo.toml
```

* tiny-keccack does have a homepage attribtute that point at the github repo. Perhaps a fallback would be viable?
* https://lib.rs/crates/crunchy does find the repo and link it. I have absolutely no idea how as the info is completely missing.

Neither of them have tagged releases, so that doesn't help very much though.

**Comment by paolobarbolini** [comment]
> https://lib.rs/crates/crunchy does find the repo and link it. I have absolutely no idea how as the info is completely missing.

I've asked on Mastodon and I was told they check if `https://github.com/{owner_name}/{crate_name}` exists

**Comment by VorpalBlade** [comment]
That is a wonderful hack. But, I would be reluctant to include such things myself if I were you.

**Comment by paolobarbolini** [comment]
> But, I would be reluctant to include such things myself if I were you.

Yeah I agree. I'd much prefer fixing the problem by sending PRs fixing Cargo.toml metadata and then have a local override to be in the meantime be able to scan the crate anyway.
